### ISSUE TITLE
Load(<lock/0000000000>, 0, 0) returned error

### ISSUE THREAD

**Issue by atomi** [restic/restic#1641]
restic version
```
restic 0.8.2 (v0.8.2-34-gaef5e037)
compiled with go1.10 on linux/arm
```
command
```
restic snapshots -r /home/atomisnapshots
```
output

```
enter password for repository:
password is correct
Load(<lock/0000000000>, 0, 0) returned error, retrying after 724.422099ms: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 831.100282ms: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 1.637837289s: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 1.625933127s: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 3.346018559s: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 2.659211462s: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 3.494960668s: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 9.411528659s: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 16.65432631s: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Load(<lock/0000000000>, 0, 0) returned error, retrying after 22.13328528s: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory
Fatal: unable to create lock in backend: open /home/atomi/snapshots/locks/0000000000000000000000000000000000000000000000000000000000000000: no such file or directory

```

This is a strange one. I found that downloading backup of my restic repo on another computer with an older restic version works correctly version 
```
restic 0.7.3 (v0.7.3-32-gd41dce5)
compiled with go1.9 on linux/amd64
```

I then tried replacing my /home/atomi/snapshots repo with that same backup and the error continues even after reboots.

**Comment by atomi** [comment]
I updated to the latest on my workstation
```
restic 0.8.2 (v0.8.2-34-gaef5e03)
compiled with go1.9.2 on linux/amd64
```

And the error appears now as well.

**Comment by atomi** [comment]
Okay I went and did a bisect and this is the commit that breaks/causes the error:

https://github.com/restic/restic/commit/44550a88a04f9bba528ab785b2f05b52f1726441

The previous commit works fine.

**Comment by fd0** [comment]
Huh, thanks a lot for the report and the bisect! It is very valuable to have people actually using the master branch before it is released so we catch such bugs (although this one is also included in 0.8.2) ;)

Can you please rebuild restic and create a debug log ([instructions](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#reporting-bugs)) and attach it to the issue?

I have a couple of ideas what could be going on here. Is there anything in the `locks/` directory in the repo?

**Comment by atomi** [comment]
> If you want to do a PR

It's alright. It's just a 2 word fix. I'll let you handle it. In truth, this was in part my fault for having a file in there in the first place - a sloppy workaround for syncing the locks directory to s3.

Thanks for the quick update :+1:
